lib/fly.js

```diff
@@ -42,9 +41,6 @@ function Fly(options) {
 
 	// attach files & funcs into the curr instance
 	assign(self, {
-		file: file,
-		plugins: plugins,
```

Comment:
Because we got `file = undefined` and then try to `path.dirname(*undefined*)` that throws an exception. And when we are sure that file been defined we assign these props below.
I was thinking about more elegant solution, but did not want to change defined architecture